.github/workflows/pull-request-checks.yaml

```diff
@@ -349,3 +349,24 @@ jobs:
       - uses: actions/checkout@v2
       - name: Check for unused irep ids
         run: ./scripts/string_table_check.sh
+
```

Comment:
:pick: Might be worth stating in the commit title that this is PR check. Otherwise it might be confused with the release actions when looking through the commit history later. Something like - "Add github action on PRs to check the Dockerfile"
:pick: The commit title should not end with a full stop. See point "4. Do not end the subject line with a period", here - [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)